accounting/others/reporting/customize.rst

```diff
@@ -1,25 +1,25 @@
 ==========================================================
-How to create a customized reports with your own formulas?
+How to create customized reports with your own formulas?
 ==========================================================
 
 Overview
 ========
 
 Odoo 9 comes with a powerful and easy-to-use reporting framework.
-Creating new reports (such as a tax report or a balance sheet for a
+Creating new reports (such as tax report or balance sheet for a
 specific country) to suit your needs is now easier than ever.
 
-Activate the developer mode
```

Comment:
Hi @tivisse ,

I didn't create this pull request, nor did I make the changes to customize.rst that @mart-e is referring to.

-Clark